src/components/AudioPlayer/Player/Player.component.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import Repeat from '../PlayerButtons/Repeat';
+import Rewind from '../PlayerButtons/Rewind';
+import Play from '../PlayerButtons/Play';
+import Forward from '../PlayerButtons/Forward';
+import Shuffle from '../PlayerButtons/Shuffle';
```

Comment:
Let's add all these imports into a single `index.js` instead to import them all at the time.